mne/preprocessing/ica.py

```diff
@@ -2079,3 +2079,233 @@ def _band_pass_filter(ica, sources, target, l_freq, h_freq, verbose=None):
         raise ValueError('Must specify both pass bands')
 
     return sources, target
+
+
+@verbose
+def corrmap(icas, template, threshold="auto", name="bads",
+            plot=True, inplace=False, ch_type="eeg"):
+    """Corrmap (Viola et al. 2009 Clin Neurophysiol) identifies the best group
+    match to a supplied template. Typically, feed it a list of fitted ICAs and
+    a template IC, for example, the blink for the first subject, to identify
+    specific ICs across subjects.
+
+    The specific procedure consists of two iterations. In a first step, the
+    maps best correlating with the template are identified. In the step, the
+    analysis is repeated with the mean of the maps identified in the first
+    stage.
+
+    Outputs a list of fitted ICAs with the indices of the marked ICs in a
+    specified field.
+
+    The original Corrmap website: www.debener.de/corrmap/corrmapplugin1.html
+
+    Parameters
+    ----------
+    icas : list
+        A list of fitted ICAs.
+    template : (int, int) tuple
+        Index of the list member from which the template should be obtained,
+        and the index of the IC.
```

Comment:
For the template IC? I don't see how, as there is AFAIK no convenient way to get the map of a specific IC.
Either way, this is how it is done in the original corrmap function.